client/etcd/config.go

```diff
@@ -73,12 +73,13 @@ func (c *keepAliveConfig) NewOptions() KeepAliveOptions {
 
 // Configuration is for config service client.
 type Configuration struct {
-	Zone         string                 `yaml:"zone"`
-	Env          string                 `yaml:"env"`
-	Service      string                 `yaml:"service" validate:"nonzero"`
-	CacheDir     string                 `yaml:"cacheDir"`
-	ETCDClusters []ClusterConfig        `yaml:"etcdClusters"`
-	SDConfig     services.Configuration `yaml:"m3sd"`
+	Zone              string                 `yaml:"zone"`
+	Env               string                 `yaml:"env"`
+	Service           string                 `yaml:"service" validate:"nonzero"`
+	CacheDir          string                 `yaml:"cacheDir"`
+	ETCDClusters      []ClusterConfig        `yaml:"etcdClusters"`
+	SDConfig          services.Configuration `yaml:"m3sd"`
+	WatchWithRevision int64                  `yaml:"watchWithRevision"`
```

Comment:
Do you still need this if you already have it in the per store override options?

Reply:
Yeah, some users may not use overrides and may want all stores created from the client to have the revision set. I.e. if the only way I'm consuming my store is by using `NewConfigServiceClient` and then `client.KV()`, I'm not using overrides but I still want the watch revision set.

Reply:
If we are adding the config on the client level, can we only add it here? So that we can still keep the `WatchWithRevision` field to the etcd implementations without polluting the kv.OverrideOptions interface